test/runtime/write.c

```diff
@@ -617,6 +618,59 @@ static void usage(const char *program_name)
            p);
 }
 
+static void fs_stress_test() 
+{
+    int num_files = 1000;
+    int fds[num_files];
+
+    /* Creating and writing to new files */
+    int i;
+    for (i = 0; i < num_files; i++) {
+        char buf[BUFLEN];
+        ssize_t rv;
+        char name[25];
+        sprintf(name, "fs_stress_test_%d", i);
+        int fd = open(name, O_CREAT | O_RDWR, S_IRUSR | S_IWUSR);
+        fds[i] = fd;
+        if (fd < 0) {
+            printf("open error\n");
+            perror("open");
+            exit(EXIT_FAILURE);
+        }
+
+        _READ(buf, BUFLEN);
+
+        if (rv == 0)
+            writetest_debug("empty source file\n");
+
+        if (rv >= BUFLEN)
+            rv = BUFLEN - 1;
+        buf[rv] = '\0';
+        writetest_debug("original: \"%s\"\n", buf);
+
+        _LSEEK(0, SEEK_SET);
+
+        ssize_t len = strlen(str);
+        _WRITE(str, len);
+    }
+
+    /* sync the filesystem */
+    syscall(SYS_sync); 
+
+    /* Deleting all files we just created */
+    for (i = 0; i < num_files; i++) 
+        close(fds[i]);
```

Comment:
Closing a file does not delete it

Reply:
@francescolavra we used remove() and added a check to ensure the files were properly removed. Is this synonymous to deleting a file?

Reply:
Yes, remove() can be used to delete a file, but if you check the return value of remove() in your code you will see that it's an error value for most (all?) files, so it's not being used correctly and is not actually deleting those files.